Eatery/Controllers/FilterBar.swift

```diff
@@ -63,26 +66,33 @@ class FilterBar: UIView {
             button.frame.size.height = frame.height - 20.0
             button.center.y = frame.height / 2
 
-            if index > 0 {
-                button.frame.origin.x = buttons[index - 1].frame.maxX + 10.0
-            } else {
-                button.frame.origin.x = 0.0
-            }
-
+            
             button.tag = index
             button.setBackgroundImage(UIImage.image(withColor: UIColor.white), for: .normal)
             button.setBackgroundImage(UIImage.image(withColor: UIColor.eateryBlue), for: .highlighted)
             button.setBackgroundImage(UIImage.image(withColor: UIColor.eateryBlue), for: .selected)
             button.setTitleColor(UIColor.white, for: .selected)
             button.addTarget(self, action: #selector(buttonPressed(sender:)), for: .touchUpInside)
-
+            
             scrollView.addSubview(button)
+            button.snp.makeConstraints({ (make) in
```

Comment:
swiftier to make it a trailing closure
i.e. `button.snp.makeConstraints { make in`